wasm-runtime/polkadot/src/runtime/staking.rs

```diff
@@ -40,45 +41,7 @@ impl StorageVec for IntentionStorageVec {
 // - n | n > current_era(): deactivating: recently representing a validator and not yet
 //   ready for transfer.
 
-/// The length of the bonding duration in eras.
-pub fn bonding_duration() -> BlockNumber {
-	Storable::lookup_default(b"sta:loc")
-}
-
-/// The length of a staking era in sessions.
-pub fn validator_count() -> usize {
-	u32::lookup_default(b"sta:vac") as usize
-}
-
-/// The length of a staking era in blocks.
-pub fn era_length() -> BlockNumber {
-	sessions_per_era() * session::length()
-}
-
-/// The length of a staking era in sessions.
-pub fn sessions_per_era() -> BlockNumber {
-	Storable::lookup_default(b"sta:spe")
-}
-
-/// The current era index.
-pub fn current_era() -> BlockNumber {
-	Storable::lookup_default(b"sta:era")
-}
-
-/// The block number at which the era length last changed.
-pub fn last_era_length_change() -> BlockNumber {
-	Storable::lookup_default(b"sta:lec")
-}
-
-/// The balance of a given account.
-pub fn balance(who: &AccountID) -> Balance {
-	Storable::lookup_default(&who.to_keyed_vec(b"sta:bal:"))
-}
-
-/// The liquidity-state of a given account.
-pub fn bondage(who: &AccountID) -> Bondage {
-	Storable::lookup_default(&who.to_keyed_vec(b"sta:bon:"))
-}
+// TRANSACTION API
```

Comment:
Why not use modules instead of formatting comments?

Reply:
done in the latest PR